### PR TITLE
Rename transfers to payments in the webui

### PR DESCRIPTION
**Pull request by kelsos**

### CODE REVIEW

File: raiden/ui/web/src/app/components/channel-table/channel-table.component.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
     </div>
     <p-dataTable [value]="channels" [paginator]="true" [rows]="10" [resizableColumns]="true" [globalFilter]="gb"
                  dataKey="channel_identifier" [rowTrackBy]="rowTrackBy">
-      <p-column field="channel_identifier" header="Channel" [sortable]="true">
+      <p-column field="channel_identifier" header="Channel" [sortable]="true" [style]="{width: '8em'}">
```
Comment (andrevmatos) [inline raiden/ui/web/src/app/components/channel-table/channel-table.component.html]: Not important, but just a headsup that, for styling single properties, you can do something like `[style.width.em]="8"`, or of course, as a static property, declare it statically too, to leave it out of the changes detectors and rendering engine.

File: raiden/ui/web/src/app/components/channel-table/channel-table.component.ts [hunk 2]
```diff
@@ -135,15 +135,17 @@ export class ChannelTableComponent implements OnInit, OnDestroy {
...
         switch (this.action) {
-            case 'transfer':
+            case 'pay':
                 if ('target_address' in response && 'identifier' in response) {
                     this.sharedService.msg({
-                        severity: 'info', summary: this.action,
-                        detail: `A transfer of amount ${response.amount} is successful with the partner ${response.target_address}`
+                        severity: 'info',
+                        summary: 'Payment',
+                        detail: `A payment of ${response.amount} was successfully send to the partner ${response.target_address}`
```
Comment (andrevmatos) [inline raiden/ui/web/src/app/components/channel-table/channel-table.component.ts]: `s/send/sent/`

File: raiden/ui/web/src/app/components/channel-table/channel-table.component.html (hunk 1 above)
Comment (kelsos) [reply raiden/ui/web/src/app/components/channel-table/channel-table.component.html]: I did try it after reading the comment and `[style.width.em]="8"` doesn't seem to have the same effect as `[style]="{width: '8em'}` for some reason.
I will leave it as it is for the moment and re-evaluate in the future.

**Comment by LefterisJP** [comment]
@kelsos can you handle the PR review comments?

**Pull request merged**